### PR TITLE
Added schedule for MM notification and Official Web News

### PR DESCRIPTION
**Pull request by IvanRuzavin**

### CODE REVIEW

File: .github/workflows/createWebNews.yaml [hunk 1]
```diff
@@ -11,6 +11,9 @@ on:
             - Official
           default: "Beta"
 
+  schedule:
```
Comment (StrahinjaJacimovic) [inline .github/workflows/createWebNews.yaml:14]: Schedule is OK, but how do we choose if it's BETA or LIVE?

A better approach would be to release BETA @10 for example, followed by LIVE news release @11.

The release @11 can be done if spreadsheet text for that date has a specific text (confirmed/released whatever).

File: .github/workflows/notify.yaml [hunk 2]
```diff
@@ -16,6 +16,9 @@ on:
         description: Notify Boards/Cards
         default: false
 
+  schedule:
```
Comment (StrahinjaJacimovic) [inline .github/workflows/notify.yaml:19]: Schedule is OK, but how do we select the values here?

Which values shall be passed to script?

**Review by StrahinjaJacimovic — CHANGES_REQUESTED** [review]

File: .github/workflows/createWebNews.yaml (hunk 1 above)
Comment (IvanRuzavin) [reply .github/workflows/createWebNews.yaml:14]: Logic now is that we check if status column has "Released" in it as by 10AM all releases should be done already. If it has it, then we proceed with Web Post to Beta, if it doesn't - we notify SDK channel to check the spreadsheet/calendar.

File: .github/workflows/notify.yaml (hunk 2 above)
Comment (IvanRuzavin) [reply .github/workflows/notify.yaml:19]: SDK Message posting is not called during scheduled run, for Boards/Cards Message we don't need any parameters.

File: .github/workflows/createWebNews.yaml [hunk 3]
```diff
@@ -3,16 +3,22 @@ name: Post Web News
...
+        type: choice
+        description: Post News on Beta or Official Website?
+        options:
+          - Beta
+          - Official
+        default: "Beta"
+
+  schedule:
+    # Schedule for posting to Beta Website at 10:00 AM
+    - cron: "0 9 * * 1-5"
```
Comment (StrahinjaJacimovic) [inline .github/workflows/createWebNews.yaml:16]:
```suggestion
    - cron: "30 9 * * 1-5"
```

**Review by StrahinjaJacimovic — APPROVED** [review]

**Pull request merged**